### PR TITLE
skip location/city for minors

### PR DESCRIPTION
**Pull request by DevChima**

### CODE REVIEW

File: go-app-ussd_tb_check.js [hunk 1]
```diff
@@ -725,35 +728,43 @@ go.app = function () {
     self.add("state_submit_data", function (name, opts) {
       var answers = self.im.user.answers;
       var msisdn = utils.normalize_msisdn(self.im.user.addr, "ZA");
+      var payload = {
+        data: {
+          msisdn: msisdn,
+          source: "USSD",
+          language: answers.state_language,
+          province: answers.state_province,
+          //city: answers.state_city,
```
Comment (erikh360) [inline go-app-ussd_tb_check.js]: Should these be commented out? We can probably just remove

**Review by erikh360 — APPROVED** [review]
looks good, just the one comment

File: go-app-ussd_tb_check.js (hunk 1 above)
Comment (DevChima) [reply go-app-ussd_tb_check.js]: I needed to run another test after removing that line in code. I'll correct.

**Review by erikh360 — APPROVED** [review]
🍰

**Pull request merged**